### ISSUE TITLE
Catkinize pr2_kinematics

### ISSUE THREAD

**Issue by ahendrix** [PR2/pr2_kinematics#2]
Catkinize and release pr2_kinematics. Depends on catkinizing and releasing:
- [ ] arm_navigation_msgs, collision_space, planning_environment, planning_models, kinematics_base, kinematics_msgs (from arm_navigation)

arm_navigation is no longer maintained; this stack may need to be dropped

**Comment by isucan** [comment]
Austin, pr2_kinematics is actually included in moveit, so when we drop arm_navigation, we will drop that too. We can still release stacks with the old build system in Hydro, which is currently planned to be the last release to include arm_navigation

**Comment by ahendrix** [comment]
My goal is to catkinize all of the PR2 stacks before releasing them into Hydro. It isn't required, but it allows PR2 users to use either build system for new projects.

What will it take to move the existing users of pr2_kinematics (pr2_arm_move_ik) to the use MoveIt instead? Are there tutorials for this sort of thing?

**Comment by isucan** [comment]
No tutorials yet, but the functionality exists with almost the same message format, so I expect it would be easy to get that done.

**Comment by UltronDestroyer** [comment]
@ahendrix 
I've decided to go ahead and move this forward with the pr2_arm_kinematics that is used in MoveIt. (See: Releasing this with the MoveIt backend should make users who build on this package happy. I've also deprecated the pr2_arm_constraints_aware package as I didn't see what it was used for and was almost entirely dependent on arm_navigation which has been deprecated. 

Any concerns/questions?
